File: Netbeans/Java/Lab6/src/model/MaxValue.java
package model;

import java.util.Arrays;



public class MaxValue {
    
    public static char maxValue(String line) {

//        View.writeLine("\tInput number: ");
//        String line = Input.string();
        char[] massive = line.toCharArray();
        Arrays.sort(massive);

//        View.writeLine("\tMax value is: " + massive[massive.length - 1]);
        return massive[massive.length - 1];
    }
    
}

File: Csharp/Laba6/Laba6/Model/MaxValue.cs
using System;

namespace Laba6
{
	public class MaxValue
	{

		public static char MaxValueFounder(int li) {

			String line = li.ToString();
			char[] massive = line.ToCharArray();
			Array.Sort(massive);
			return massive[massive.Length - 1];

		}

	}
}

